lib/stat_tracker.rb

```diff
@@ -23,6 +23,49 @@ def initialize(files)
     end
   end
 
+  def all_season_game_id(season)
+    @games.map do |game|
+      game.game_id if game.season == season
+    end.compact 
```

Comment:
nice helper method, we can utilize and build off that for the rest of the methods.